<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MigrantMainWindow</class>
 <widget class="QMainWindow" name="MigrantMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>356</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MigrantMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="edit_tab">
       <attribute name="title">
        <string>Ввод данных</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="fieldsBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>329</width>
             <height>256</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Панель</string>
   </property>
   <property name="whatsThis">
    <string>Панель</string>
   </property>
   <property name="accessibleName">
    <string>Панель</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <widget class="QMenu" name="menuPrint">
     <property name="title">
      <string>Пе&amp;чать</string>
     </property>
     <addaction name="actionPrint1"/>
     <addaction name="actionPrint2"/>
     <addaction name="actionPrintDual"/>
     <addaction name="actionPrint"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="menuPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>В&amp;ид</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionFit_width"/>
    <addaction name="separator"/>
    <addaction name="action_Panel"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>Созд&amp;ать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Сохранить &amp;как</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>В&amp;ыход</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О &amp;программе</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>У&amp;величить</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>У&amp;меньшить</string>
   </property>
  </action>
  <action name="actionFit_width">
   <property name="text">
    <string>По &amp;ширине</string>
   </property>
  </action>
  <action name="action_Panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Панель &amp;инструментов</string>
   </property>
  </action>
  <action name="actionPrint1">
   <property name="text">
    <string>Печать лицевой стороны</string>
   </property>
  </action>
  <action name="actionPrint2">
   <property name="text">
    <string>Печать оборотной стороны</string>
   </property>
  </action>
  <action name="actionPrintDual">
   <property name="text">
    <string>Двусторонняя печать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Просмотр печати</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MigrantMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
